src/haxelib/client/Main.hx

```diff
@@ -1037,17 +1048,17 @@ class Main {
 		}
 		else
 		{
-			var inf = try site.infos(p) catch( e : Dynamic ) { Sys.println(e); return; };
-			p = inf.name;
```

Comment:
This one was needed, actually, see f358bfada2812970f94986defe6f2108d232dfe2, gotta look into how to get that name in proper case without `infos` call.